contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -227,6 +227,30 @@ BEGIN
 END
 $$;
 
+CREATE OR REPLACE PROCEDURE sys.analyze_babelfish_catalogs()
```

Comment:
Drop this procedure once we are done with the usage.

Reply:
It will get used during each upgrade so no need to create and drop everytime.